lib/commands/logs.js

```diff
@@ -0,0 +1,41 @@
+const { api, HOST, WS_HOST } = require('../util/api');
+const open = require('open');
+const WebSocket = require('ws');
+
+const getApps = () => {
+  return new Promise((resolve, reject) => {
+    api.get(HOST + '/api/apps').then((response) => {
+      resolve(response.data.apps);
+    }).catch((error) => {
+      reject(error);
+    });
+  })
+};
+
+const followLogs = (appId) => {
+  const ws = new WebSocket(`${WS_HOST}/app-logs?appId=${appId}`);
```

Comment:
Send this request to `/api/app-logs` that way the server knows to authenticate using JWT instead of cookies (see my recent pull request on the PaaS repo).